pkgs/development/python-modules/keyutils/default.nix

```
@@ -16,7 +16,12 @@ buildPythonPackage rec {
     substituteInPlace setup.py --replace '"pytest-runner"' ""
   '';
 
+  postConfigure = ''
+    cython keyutils/_keyutils.pyx
+  '';
```

```suggestion
  postBuild = ''
    cython keyutils/_keyutils.pyx
  '';
```

`preBuild` or `postBuild` make more sense to me, since cython represents a build process.

---

Testing this one right now as the code is already there and just needs to be re-generated properly; this would be fixed upstream but that project has been archived and as mentioned in #263482 there's no fork. Will fix to `preBuild` if need be.

---

Yeah, same error with `postBuild`, so changing to `preBuild`.

Looks like there's some errors with the test now, will grok and see what we need to do (in ref to https://github.com/NixOS/nixpkgs/pull/263500#discussion_r1373690605).